addons/im_livechat/views/im_livechat_chatbot_templates.xml

```diff
@@ -4,22 +4,23 @@
         <template id="chatbot_test_script_page">
             <t t-call="web.frontend_layout">
                 <t t-set="no_livechat" t-value="True"/>
+                <t t-set="chatbot_name" t-value="storeData['chatbot.script'][0]['name']" />
```

Comment:
Store data should ideally not be introspected like that, only used to give to `store.insert()`.

If we need data in a different situation, it's best to return a separate key. In particular, it makes no sense to give store data to a python template.

`chatbot_test_script_page` should only be rendered with the specific keys that it needs.